Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -1189,18 +1199,18 @@ private bool TryUpdateGrabVirtualItems(Entity<PullerComponent> puller, Entity<Pu
     /// </summary>
     /// <param name="pullable">Grabbed entity</param>
     /// <returns></returns>
-    private bool AttemptGrabRelease(Entity<PullableComponent?> pullable)
+    private GrabResistResult AttemptGrabRelease(Entity<PullableComponent?> pullable)
     {
         if (!Resolve(pullable.Owner, ref pullable.Comp)
             || _timing.CurTime < pullable.Comp.NextEscapeAttempt)
-            return false;
+            return GrabResistResult.TooSoon;
 
         if (_random.Prob(pullable.Comp.GrabEscapeChance))
-            return true;
+            return GrabResistResult.Succeeded;
 
-        pullable.Comp.NextEscapeAttempt = _timing.CurTime.Add(TimeSpan.FromSeconds(3));
+        pullable.Comp.NextEscapeAttempt = _timing.CurTime.Add(TimeSpan.FromSeconds(2));
```

Comment:
```suggestion
        pullable.Comp.NextEscapeAttempt = _timing.CurTime.Add(TimeSpan.FromSeconds(pullable.Comp.EscapeAttemptCooldown));
```
Just add this to the component instead of sticking it in the system, silly.

Reply:
I wanted to, but I am clueless, so I ended up just changing the value where it was originally.

Reply:
Okay I think it is done